Cargo.toml

```diff
@@ -210,6 +210,265 @@ runtime-metrics = [ "polkadot-cli/runtime-metrics" ]
 pyroscope = ["polkadot-cli/pyroscope"]
 jemalloc-allocator = ["polkadot-node-core-pvf-worker/jemalloc-allocator", "polkadot-overseer/jemalloc-allocator"]
 
+[patch."https://github.com/paritytech/substrate"]
+node-template = { path = "/Users/gav/Core/substrate/bin/node-template/node" }
```

Comment:
You can revert this, the merge bot will then update the lock (it is expected not to build here).